---
string vol_water_reading
string temperature_reading
string conductivity_reading